sdk/textanalytics/Azure.AI.TextAnalytics/tests/SessionRecords/RecognizePiiEntitiesTests/RecognizePiiEntitiesBatchConvenienceTest.json

```diff
@@ -1,23 +1,23 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://ta-s-westeurope.cognitiveservices.azure.com/text/analytics/v3.1-preview.4/entities/recognition/pii?showStats=false\u0026domain=phi\u0026stringIndexType=Utf16CodeUnit",
+      "RequestUri": "https://mariari-westus2-s.cognitiveservices.azure.com/text/analytics/v3.1-preview.4/entities/recognition/pii?showStats=false\u0026domain=phi\u0026stringIndexType=Utf16CodeUnit",
```

Comment:
looking at the recording files I noticed that `domain=phi` is always set, even if the user didn't request it.
This is clearly a bug on the SDK, so I created issue https://github.com/Azure/azure-sdk-for-net/issues/19086 to fix it